Religious Heads and Holy Sites

zumbak04 opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Assigned 5 holy sites to each religion. The holy sites have their own unique bonuses.
- Added historical religious heads for:
  - Tirani
  - Ragnaric
  - Holy Light
  - Beloric
  - All Loa faiths
  - All Old Gods' faiths
  - Elunic
  - Draconic

## How to test:
Check these religious heads and holy sites in-game, tell your opinion.

TheGroxEmpire commented:
**Light Faith**

- Gilneas does not really fit as a holy site for Holy Light. Lordaeron seems to fit better

> In the early days of humanity and its civilization, many tribes of humans had primitive belief systems that incorporated simple nature magic. However, the rise of organized religion such as the Holy Light and the potent arcane magics introduced by the high elves quickly supplanted such traditions. Gilneas, due to its relative isolation, has retained a degree of their ancient culture in the contemporary era. The religious leaders of what was in Gilneas referred to as the "old ways" eventually became "harvest-witches" (https://wow.gamepedia.com/Ask_CDev#Ask_CDev_Answers_-_Round_3)

- The Naarist and Lightbound should not have Lordaeron as their holy site. But I figure that it may be a placeholder

**Elunic**
- Eldre'thalas / Dire Maul was a Highborne capital and insignificant for Elunic Holy Site. I figured that Val'Sharah has not been implemented yet, but as soon it does Temple of Elune that is located there (Tomb of Sargeras) is a clear replacement for it.

**Draconic**
- Dragonflights should not be crammed into a single umbrella religion with led by Alexstrazsa. Each dragonflight act on their own dragon aspect behalf.

**Old Gods**
- Not much I can find about Prophet Kosumoth except that it is a C'Thraxxi elite. Uu'nat the Harbinger of the Void from Crucible Storm raid imo is a better choice as religious head of N'zothi

**Loa Faiths**
- Hakkari should be led by Jammal'an the Prophet (https://wow.gamepedia.com/Jammal%27an_the_Prophet)

**Beloric**
- Title should be "Grand Magister" instead of "High Magister" (https://wow.gamepedia.com/Grand_Magister)

TheGroxEmpire reviewed (commented):
**Light Faith**

- Gilneas does not really fit as a holy site for Holy Light. Lordaeron seems to fit better

> In the early days of humanity and its civilization, many tribes of humans had primitive belief systems that incorporated simple nature magic. However, the rise of organized religion such as the Holy Light and the potent arcane magics introduced by the high elves quickly supplanted such traditions. Gilneas, due to its relative isolation, has retained a degree of their ancient culture in the contemporary era. The religious leaders of what was in Gilneas referred to as the "old ways" eventually became "harvest-witches" (https://wow.gamepedia.com/Ask_CDev#Ask_CDev_Answers_-_Round_3)

- The Naarist and Lightbound should not have Lordaeron as their holy site. But I figure that it may be a placeholder

**Elunic**
- Eldre'thalas / Dire Maul was a Highborne capital and insignificant for Elunic Holy Site. I figured that Val'Sharah has not been implemented yet, but as soon it does Temple of Elune that is located there (Tomb of Sargeras) is a clear replacement for it.

**Draconic**
- Dragonflights should not be crammed into a single umbrella religion with led by Alexstrazsa. Each dragonflight act on their own dragon aspect behalf.

**Old Gods**
- Not much I can find about Prophet Kosumoth except that it is a C'Thraxxi elite. Uu'nat the Harbinger of the Void from Crucible Storm raid imo is a better choice as religious head of N'zothi

**Loa Faiths**
- Hakkari should be led by Jammal'an the Prophet (https://wow.gamepedia.com/Jammal%27an_the_Prophet)

**Beloric**
- Title should be "Grand Magister" instead of "High Magister" (https://wow.gamepedia.com/Grand_Magister)

zumbak04 commented:
Thanks for your review!

> * Gilneas does not really fit as a holy site for Holy Light. Lordaeron seems to fit better

Changed in 7d986a75a5d80d5ec126ab04a0b3fb089b087d52.

> * The Naarist and Lightbound should not have Lordaeron as their holy site. But I figure that it may be a placeholder

It's an important holy light place. The Draenei themselves don't have many holy sites on Azeroth.

> * Eldre'thalas / Dire Maul was a Highborne capital and insignificant for Elunic Holy Site. I figured that Val'Sharah has not been implemented yet, but as soon it does Temple of Elune that is located there (Tomb of Sargeras) is a clear replacement for it.

Agree.

> * Dragonflights should not be crammed into a single umbrella religion with led by Alexstrazsa. Each dragonflight act on their own dragon aspect behalf.

More religions... may do, but later.

> * Not much I can find about Prophet Kosumoth except that it is a C'Thraxxi elite. Uu'nat the Harbinger of the Void from Crucible Storm raid imo is a better choice as religious head of N'zothi

Added in c10432a095814825cdb2886bc7baa688845e253c.

> * Hakkari should be led by Jammal'an the Prophet (https://wow.gamepedia.com/Jammal%27an_the_Prophet)

Added in c10432a095814825cdb2886bc7baa688845e253c.

> * Title should be "Grand Magister" instead of "High Magister" (https://wow.gamepedia.com/Grand_Magister)

Changed in e47232fbc13f1e09487e20229385669d3d588e06.

TheGroxEmpire reviewed (commented):
- Move Goldrinism holy site from Timbermaw to Shrine of Goldrinn in Jillaei, Hyjal. Jillaei county faith is conveniently Goldrinism already.
<details>

![image](https://user-images.githubusercontent.com/47816228/98815518-d4741a00-2459-11eb-987d-57877a279104.png)
![image](https://user-images.githubusercontent.com/47816228/98815625-071e1280-245a-11eb-89b1-a67557993a72.png)
</details>

- Jammal'an and Uu'nat doesn't appear on year 583 start but correctly appear on other start

zumbak04 commented:
> * Move Goldrinism holy site from Timbermaw to Shrine of Goldrinn in Jillaei, Hyjal. Jillaei county faith is conveniently Goldrinism already.

Done in 50504458def3ce4a7718b5e919878529ca1d36b9.

> * Jammal'an and Uu'nat doesn't appear on year 583 start but correctly appear on other start

Jammal'an is too young to rule. Uu'nat is a naga. The nagas first appeared after the year 603.

TheGroxEmpire reviewed (commented):
> Jammal'an is too young to rule. Uu'nat is a naga. The nagas first appeared after the year 603.

Naga has been existing since the sundering as per chronicle Volume 1 page 105. 603 (Reign of Chaos) was when they first show themselves to the surface dwellers.
<details>

![image](https://user-images.githubusercontent.com/47816228/98823410-a2b48080-2464-11eb-82a9-72c1ffd41288.png)
</details>
> When she struck her bargain with N'Zoth ten thousand years ago, Queen Azshara offered one of her most devoted followers into the Old God's service. Now Uu'nat and the crawling masses he commands herald the coming of their master, as foretold by the stars. (https://wow.gamepedia.com/Uu%27nat)

zumbak04 commented:
> Naga has been existing since the sundering as per chronicle Volume 1 page 105. 603 (Reign of Chaos) was when they first show themselves to the surface dwellers.

That's what I meant.

MemeWizard commented on common/religion/holy_sites/wc_holy_sites.txt:
```diff
@@ -0,0 +1,1326 @@
+	}
+}
+#Shrine of Goldrinn
+jillaei = {
```
no loc

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/99657760-a1d8ab80-2a6f-11eb-8459-4a3d9ddc3c4c.png)

</details>

zumbak04 replied:
Done in 4475f275b0c9e0328f98c68f84cf628f01f3fc7e.

MemeWizard commented on common/religion/holy_sites/wc_holy_sites.txt:
```diff
@@ -0,0 +1,1326 @@
+	character_modifier = {
+		name = holy_site_astranaar_effect_name
+		stewardship = 2
+		faith_creation_piety_cost_mult = 0.3
```
It's weird debuff.
So, player need to lose this province if they want to reform religion?

Btw, if this is mistake, then -30% reformation cost is the most weird buff I have seen. I suggest you to replace it with something else, otherwise conquering this holy site will make faith more venerable to new heresies.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/99657934-d9dfee80-2a6f-11eb-8646-4579fc0f09fd.png)

</details>

zumbak04 replied:
> Btw, if this is mistake, then -30% reformation cost is the most weird buff I have seen. I suggest you to replace it with something else, otherwise conquering this holy site will make faith more vulnerable to new heresies.

I don't agree. It makes religion more flexible. A very good buff if the religion is planned to be reformed. It will allow you to pick stronger tenets and doctrines for a lower price.

zumbak04 replied:
Made it negative in 7f4a35af65c3e0a323ee5b05a28ead053b5ed94f.

MemeWizard replied:
> It makes religion more flexible.

> It will allow you to pick stronger tenets and doctrines for a lower price.

I do not disagree with that. In fact, you just said the same thing as me, except you tried to make it sounds like a good buff, which is it not, if you don't plan to create a new heresy.

> A very good buff if the religion is planned to be reformed.

Only NE and their heresies will get this buff, since only they use this holy site.

So, the plan is to make Elunic religion the easiest to make heresies from?

zumbak04 replied:
> I do not disagree with that. In fact, you just said the same thing as me, except you tried to make it sounds like a good buff, which is it not, if you don't plan to create a new heresy.

It's a great buff, but it's situative. I would like to have it in 75% of my playthroughs.

> So, the plan is to make Elunic religion the easiest to make heresies from?

I think so. It makes sense since the night elves changed a lot after the Third War and the War of the Ancients

MemeWizard replied:
> It's a great buff, but it's situative. I would like to have it in 75% of my playthroughs.

Idk, personally, I wouldn't want for any characters of my faith to make new heresies. Especially when my character makes custom religion and tries to spread it as much as possible. Good thing that AI doesn't do it, I guess?

> I think so. It makes sense since the night elves changed a lot after the Third War and the War of the Ancients

How much, tho?
They outlawed arcane magic after the War of the Ancients (So, in-game terms, they changed one criminal doctrine).
Aren't they just allowed males to be priests and females to be druids after the Third War (So, in-game it will only change female priesthood doctrine to equal)?
Changing 2 doctrines in 10 000 years doesn't sound like "changed a lot", tbh.

Doctrines can be changed via events and decisions, like how Christian denominations can lose or gain ecumenism doctrine in vanilla, so I don't think it's a good idea to suggest players make a whole new religion instead of giving them a decision to change just one doctrine.

zumbak04 replied:
> Idk, personally, I wouldn't want for any characters of my faith to make new heresies. Especially when my character makes custom religion and tries to spread it as much as possible. Good thing that AI doesn't do it, I guess?

Convert them back or size titles (if you really need). If you fear heresies, you shouldn't do holy wars and be wide. That's the main things that push heresies. Of course, you can punish people for picking Scholar focus, but I don't think it's worth it.
Basically, don't think it's bad to have a few religions in your realm especially if they're close. Together you can holy war everyone more often because you don't share the same fervour.

> How much, tho?
> They outlawed arcane magic after the War of the Ancients (So, in-game terms, they changed one criminal doctrine).

And allowed it back around Cata.

> Aren't they just allowed males to be priests and females to be druids after the Third War (So, in-game it will only change female priesthood doctrine to equal)?

> Changing 2 doctrines in 10 000 years doesn't sound like "changed a lot", tbh.

Before the Third War, they may have Fundamentalism doctrine. They didn't allow other races in their society and were isolationists.

Before the War of the Ancients, they may have no Sanctity of Nature tenet (they rediscovered Cenarius when Malfurion became the first druid). Also, they may be a different faith of Elunism on the Broken Isles without this tenet (the nightborne), but this requires more research.

MemeWizard commented on common/religion/holy_sites/wc_holy_sites.txt:
```diff
@@ -0,0 +1,1326 @@
+	character_modifier = {
+		name = holy_site_lordaeron_effect_name
+		diplomacy_per_piety_level = 1
+		monthly_piety_gain_per_knight_add = 0.05
```
Don't work

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/99658649-d862f600-2a70-11eb-9844-698b812d1d69.png)

</details>

zumbak04 replied:
Changed in 0b8e129ed81ceb91b14b5b7a7c20e749925434a3.

MemeWizard commented on common/religion/holy_sites/wc_holy_sites.txt:
```diff
@@ -0,0 +1,1326 @@
+	
+	character_modifier = {
+		name = holy_site_hearthglen_effect_name
+		holy_order_government_opinion = 10
```
no loc

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/99658732-f4669780-2a70-11eb-8167-115a214a02dc.png)

</details>

zumbak04 replied:
Done in 33a6888aea847134660e8e12054a2883e358f060.

MemeWizard commented on common/religion/holy_sites/wc_holy_sites.txt:
```diff
@@ -0,0 +1,1326 @@
+	
+	character_modifier = {
+		name = holy_site_gundrak_effect_name
+		church_holding_holding_build_gold_cost = -0.4
```
Ok, but imo, that's A LOT

zumbak04 replied:
Changed in ce946f28151879466b666481478893f1d8abeb9c.

MemeWizard commented on common/religion/holy_sites/wc_holy_sites.txt:
```diff
@@ -0,0 +1,1326 @@
+	
+	character_modifier = {
+		name = holy_site_icemist_effect_name
+		army_maintenance_mult = -0.01
```
Honestly, -1% sounds like a joke.

zumbak04 replied:
Changed in afec05ea2d8b522d7578c3782f518813f1ee6ec4.

MemeWizard commented on common/religion/religions/shathgral_group.txt:
```diff
@@ -289,7 +322,14 @@
+			religious_head = d_ghuun_worship
 			icon = custom_faith_1
+
+			holy_site = ahnqiraj
+			holy_site = zulnazman
+			holy_site = ulduar
+			holy_site = storm_isle
+			holy_site = lulkail
```
I suggest replacing 3 of them with 3 seals of g'huun

zumbak04 replied:
Changed holy sites in 65ebb00e59743b59df66b529d72661a0d6133aa1.

MemeWizard commented on common/religion/religions/shathgral_group.txt:
```diff
@@ -267,7 +293,14 @@
+			religious_head = d_yshaarj_worship
 			icon = custom_faith_1
+			
+			holy_site = ahnqiraj
+			holy_site = zulnazman
+			holy_site = ulduar
+			holy_site = storm_isle
+			holy_site = lulkail
```
I think it will be fine if Y'Shaarj will have more holy sites (7 sha locations), just like vanilla have 7 holy sites for some religions; Instead of what we have now - the same 5 sites for every shathgral faith

zumbak04 replied:
These locations in WoW aren't tied to important sites. Usually, it's just a place where someone got pissed off and summoned a big sha.

Changed holy sites in 65ebb00e59743b59df66b529d72661a0d6133aa1.

zumbak04 commented on common/religion/holy_sites/wc_holy_sites.txt:
```diff
@@ -0,0 +1,1326 @@
+	character_modifier = {
+		name = holy_site_draktharon_effect_name
+		dread_gain_mult = 0.3
+		archers_pursuit_mult = 0.1
```
Done in 0951dab136f90f16f6aded6fac89e0cc3de53aae.

MemeWizard replied:
Oh, btw, generic archers retinue don't have pursuit stat at all...

zumbak04 replied:
Changed in 41de90115c9c4582d8cd7c79aaaf57bf56a7871e

MemeWizard commented on common/religion/holy_sites/wc_holy_sites.txt:
```diff
@@ -0,0 +1,1356 @@
+### ordos_cult
+#xuen
+#niuzao
+osul = {
```
no loc

![image](https://user-images.githubusercontent.com/46576860/99810284-8eebd700-2b54-11eb-92aa-990b6fc1dd6b.png)

zumbak04 replied:
Changed in 41de90115c9c4582d8cd7c79aaaf57bf56a7871e.

MemeWizard commented on common/religion/holy_sites/wc_holy_sites.txt:
```diff
@@ -0,0 +1,1356 @@
+	
+	character_modifier = {
+		name = holy_site_darnassus_effect_name
+		forest_cancel_negative_supply = 1
```
Revert changes in that line from ce946f28151879466b666481478893f1d8abeb9c

```suggestion
		forest_cancel_negative_supply = yes
```

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/99810462-c8244700-2b54-11eb-8daa-dc832511aeae.png)

</details>
